cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -41,32 +68,35 @@ if (NOT SNAPPY_FOUND)
     LOG_BUILD 1
     LOG_INSTALL 1
     BUILD_BYPRODUCTS "${SNAPPY_STATIC_LIB}")
+
+  set (SNAPPY_VENDORED TRUE)
 endif ()
-include_directories (SYSTEM ${SNAPPY_INCLUDE_DIRS})
+
+include_directories (SYSTEM ${SNAPPY_INCLUDE_DIR})
 add_library (snappy STATIC IMPORTED)
 set_target_properties (snappy PROPERTIES IMPORTED_LOCATION ${SNAPPY_STATIC_LIB})
-set (SNAPPY_LIBRARIES snappy)
-add_dependencies (snappy snappy_ep)
-install(DIRECTORY ${SNAPPY_PREFIX}/lib DESTINATION .
```

Comment:
AFAIK `cpack` depends on `install` as well to make a package. So I would vote for your other option of using `INSTALL_THIRDPARTY_LIBS=on`.

Reply:
Done. Went with `INSTALL_VENDORED_LIBS`, which defaults to on.